documentation/release_notes.markdown

```diff
@@ -23,45 +24,64 @@ Puppet Server 2.1 ships for the following supported platforms:
+
+### Misc Bug Fixes
+* [SERVER-157](http://tickets.puppetlabs.com/browse/SERVER-157) - Utilize keylength puppet conf value for generating SSL cert keys.
+* [SERVER-273](http://tickets.puppetlabs.com/browse/SERVER-273) - Upgrade to JRuby 1.7.19 / fix for jruby-pool DELETE memory leak.
+* [SERVER-345](http://tickets.puppetlabs.com/browse/SERVER-345) - Fixup usages of cacert / localcacert in master.
+* [SERVER-404](http://tickets.puppetlabs.com/browse/SERVER-404) - Properly create /var/run/puppetserver dir at service startup.
+* [SERVER-442](http://tickets.puppetlabs.com/browse/SERVER-442) - Fix for a problem where `file_metadatas` requests to the master which include multiple `ignore` parameters were being mishandled. This had previously led to an agent downloading files from the master which should have been ignored.
+* [SERVER-535](http://tickets.puppetlabs.com/browse/SERVER-535) - Tests in the `jruby-puppet-service-test` namespace were using an invalid configuration map, which caused them to throw an error.
+* [SERVER-541](http://tickets.puppetlabs.com/browse/SERVER-541) - Disabled the display of verbose output that appeared during a package upgrade.
+* [SERVER-564](http://tickets.puppetlabs.com/browse/SERVER-564) - Re-enabled the master `status` endpoint
+* [SERVER-647](http://tickets.puppetlabs.com/browse/SERVER-647) - Puppet Server failed to start with with an uncaught exception when master-{code, run, log}-dir settings were not defined. This patch addresses the problem by setting the values to nil, which will cause Puppet Server to consult Puppet for the values.
+* [SERVER-657](http://tickets.puppetlabs.com/browse/SERVER-657) - Fixed `puppetserver foreground` on Ubuntu.
+* [SERVER-655](http://tickets.puppetlabs.com/browse/SERVER-655) - Upgrade to JRuby 1.7.20 caused server fail loading the Bouncy Castle jar.
+* [SERVER-659](http://tickets.puppetlabs.com/browse/SERVER-659) - Restored broken http client timeout settings.
+* [SERVER-682](http://tickets.puppetlabs.com/browse/SERVER-682) - Fixed an issue where logback levels weren’t changed unless you restarted Puppet Server. This functionality had been provided in Puppet Server 1.0.2 but was inadvertently removed in Puppet Server 1.0.8, then merged with 2.0.
+* [SERVER-683](http://tickets.puppetlabs.com/browse/SERVER-683) - Changed the logic in the `legacy-routes-service` to get the route of the master-service via the service protocol name rather than by a hard-coded service name. This allows for the `legacy-routes-service` to pull the route from whatever service implementing the MasterService protocol happens to be in the service stack, i.e., master-service for open source Puppet or pe-master-service for PE.
+* [SERVER-684](http://tickets.puppetlabs.com/browse/SERVER-684) - Append `source_permissions=use` to 3.x file_metadata requests.
+* [SERVER-685](http://tickets.puppetlabs.com/browse/SERVER-685) - Investigated `environment_scenario` test failures.
```

Comment:
I'd remove the note about SERVER-685.  Mostly a housekeeping ticket and doesn't affect the Puppet Server 2.1 release directly.